docs/json/guide-only/language-german-and-original.json

```
@@ -0,0 +1,38 @@
+{
+  "trash_id": "guide-only",
+  "trash_scores": {
+    "default": -10000
+  },
+  "trash_description": "Language: German Dual Language and Multi-language releases",
```

```suggestion
  "trash_description": "Language: German Dual and Multi language releases",
```
I think wording this like this is better